crates/pixi-build-backend/src/compilers.rs

```diff
@@ -40,3 +44,70 @@ pub fn default_compiler(platform: Platform, language: &str) -> Option<String> {
         .to_string(),
     )
 }
+
+pub fn compiler_requirements(language: &str) -> Vec<Item<PackageDependency>> {
+    match language {
+        "fortran" => vec!["gfortran".parse().unwrap()],
+        lang if !["c", "cxx"].contains(&lang) => vec![language.parse().unwrap()],
+        _ => {
```

Comment:
Lets swap these match statements. As I read it, any language that is not "c" or "cxx" takes the first match, while the rest (`c` or `cxx` fall through). Lets instead write this as:

```suggestion
        "c"|"cxx" => { ... }
        _ => vec![language.parse().unwrap()],
```

Reply:
This function could also benefit from a distinct unit test.